app/src/main/java/org/gdg/frisbee/android/activity/GdgActivity.java

```diff
@@ -172,17 +172,17 @@ protected void onActivityResult(int requestCode, int responseCode, Intent intent
                         // If the error resolution was successful we should continue
                         // processing errors.
                         mSignInProgress = STATE_SIGN_IN;
+
```

Comment:
This fixes the infinite loop, or at least it does not try to connect when the user presses "Cancel"